ynput/ayon-core
# Chore: Move RoyalRender client code

## Changelog Description
Moved RoyalRender client code next to server codebase.

## Testing notes:
1. Create package of royalrender and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. RoyalRender integration should work as before.
